<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> 
background-color: rgb(112, 156, 231);
font: 700 15pt &quot;Sans&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="QVBoxLayout">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color:   rgb(163, 193, 244);
    border-radius: 20px;</string>
    </property>
    <property name="title">
     <string>                                        textEditor</string>
    </property>
    <widget class="QTextEdit" name="textWidget">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>100</y>
       <width>421</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
background-color: rgb(255, 255, 255)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="activerButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color:rgb(0, 0, 0);
font: 700 12pt &quot;Sans&quot;;</string>
     </property>
     <property name="text">
      <string>activer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="desactiverButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>250</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color:rgb(0, 0, 0);
font: 700 12pt &quot;Sans&quot;;</string>
     </property>
     <property name="text">
      <string>descativer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
